### PR TITLE
templates: Added github templates for Issues and PRs as well as contributing guide.

### PR DESCRIPTION
**Pull request by Bplotka**
PTAL @rackonnoiter

Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by domgreen** [comment]
Would get them to say what version of go, prom and thanos they are using .

**Comment by Bplotka** [comment]
It is all there, but forgot about go version, good point

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+**Thanos, Prometheus and Golang version used**
+
+<!--
+Output of "thanos --version" or docker image:tag used.
+(Double-check if all deployed components/services have expected versions)
+
+If you are using custom build from master branch, have you checked out the tip of the master?
+-->
+
+**What happened?**
```
Comment (stefan-improbable) [inline .github/ISSUE_TEMPLATE.md]: Remove the question marks everywhere

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!--
+    Keep PR title verbose enough and add prefix telling
+    about what components it touches e.g "query:" or ".*:"
+-->
+
+## Changes
+
+<!-- Enumerate changes you made -->
+
+## Verify
```
Comment (stefan-improbable) [inline .github/PULL_REQUEST_TEMPLATE.md]: "## Verification" would be better

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+First of all, thank you for helping us, improve the Thanos!
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: remove this line

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+When contributing not obvious change to Thanos repository, please first
+discuss the change you wish to make via issue or slack, or any other
+method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: Please follow the (code of conduct)(link) in all your interactions with the project.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+First of all, thank you for helping us, improve the Thanos!
+
+When contributing not obvious change to Thanos repository, please first
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: Please first discuss any non obvious changes via issue, slack or other method with the owners of this repository before making a change.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+discuss the change you wish to make via issue or slack, or any other
+method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Follow [this](docs/getting_started.md) to prepare thanos. Be familiar with Makefile commands, since they can be useful.
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: 1. Read (getting started docs)(docs/getting_started.md)
2.  Familiarize yourself with Makefile commands like `format`, `build`, ...

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+Check Makefile itself for commands like `format`, `build`, `proto` and `test`.
+2. Keep PRs as small as possible. Chain them if needed (base PR on other PRs).
+3. If you don't have live object store ready add these envvars to skip tests for these:
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: If you don't have *a* live object store

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+- THANOS_SKIP_GCS_TESTS to skip GCS tests.
+- THANOS_SKIP_S3_AWS_TESTS to skip AWS tests.
+
+If you skip both of these, the store specific tests will be run against memory object storage only.
+CI runs GCS and inmem tests only for now. Not having these variables will produce auth errors against GCS or AWS tests.
+
+4. If your change affects users (adds or removes feature) consider adding the item to [CHANGELOG](CHANGELOG.md)
+5. You may merge the Pull Request in once you have the sign-off of at least one developers with write access, or if you
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+6. If you feel like your PR waits too long for a review, feel free to ping `#thanos-dev` channel on our slack for review!
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: hyperlink to slack

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+## Code of Conduct
```
Comment (stefan-improbable) [inline CONTRIBUTING.md]: Remove this section, the mention in the beginning of this doc is sufficient.

File: docs/getting_started.md [hunk 10]
```diff
@@ -8,7 +8,8 @@ See [this](storage.md) for up-to-date list of available object stores for Thanos
 
 ## Requirements
 
-* One or more [Prometheus](https://prometheus.io) v2.0.0 installations
+* One or more [Prometheus](https://prometheus.io) v2.2.1+ installations (strictly v2.0.0 is supported too but not recommended)
```
Comment (stefan-improbable) [inline docs/getting_started.md]: s/strictly.../v2.0.0 works too, but it is not recommended/

**Pull request merged**